Couldn't find resource eclipse-fmt.xml

Hi,

This is my first time using spotless and maven. I already following the documentation and trying to install maven on eclipse and have no problem at all with updating the project. But when i try to run the command mvn spotless:check it give me error 

> [ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:1.22.0:check (default-cli) on project idm: Execution default-cli of goal com.diffplug.spotless:spotless-maven-plugin:1.22.0:check failed: Unable to locate file with path: <path>/eclipse-fmt.xml: Could not find resource '<path>/eclipse-fmt.xml'. -> [Help 1]

I'm using maven 3.5.3 and spotless 1.22.0 and using windows 10.

What's the inside of eclipse-fmt.xml, i try to find it on your code but seems like i can't find anything.

Thank you for your help

--- a/plugin-maven/README.md
+++ b/plugin-maven/README.md
@@ -98,7 +98,7 @@ By default, all files matching `src/main/java/**/*.java` and `src/test/java/**/*
      </licenseHeader>
      <eclipse>
        <!-- Optional, otherwise Eclipse defaults are used. Eclipse preference or property files are also supported. -->
-       <file>${basedir}/eclipse-fmt.xml</file>
+       <file>${basedir}/eclipse-format.xml</file>
        <!-- Optional, available versions: https://github.com/diffplug/spotless/tree/master/lib-extra/src/main/resources/com/diffplug/spotless/extra/config/eclipse_jdt_formatter -->
        <version>4.7.1</version>
      </eclipse>
@@ -112,13 +112,13 @@ By default, all files matching `src/main/java/**/*.java` and `src/test/java/**/*
      <importOrder>
        <!-- Specify either order or file, but not both -->
        <order>java,javax,org,com,com.diffplug,</order>
-       <file>${basedir}/importOrder</file>
+       <file>${basedir}/eclipse.importorder</file>
      </importOrder>
   </java>
 </configuration>
 ```
 
-See [ECLIPSE_SCREENSHOTS](../ECLIPSE_SCREENSHOTS.md) for screenshots that demonstrate how to get and install the eclipseFormatFile and importOrderFile mentioned above.
+See [ECLIPSE_SCREENSHOTS](../ECLIPSE_SCREENSHOTS.md) for screenshots that demonstrate how to get and install the Eclipse format configuration file and Eclipse import order file mentioned above.
 
 <a name="scala"></a>
 

--- a/plugin-maven/README.md
+++ b/plugin-maven/README.md
@@ -97,6 +97,7 @@ By default, all files matching `src/main/java/**/*.java` and `src/test/java/**/*
        <file>${basedir}/license-header</file>
      </licenseHeader>
      <eclipse>
+       <!-- Optional, otherwise Eclipse defaults are used. Eclipse preference or property files are also supported. -->
        <file>${basedir}/eclipse-fmt.xml</file>
        <!-- Optional, available versions: https://github.com/diffplug/spotless/tree/master/lib-extra/src/main/resources/com/diffplug/spotless/extra/config/eclipse_jdt_formatter -->
        <version>4.7.1</version>
@@ -116,6 +117,8 @@ By default, all files matching `src/main/java/**/*.java` and `src/test/java/**/*
   </java>
 </configuration>
 ```
+
+See [ECLIPSE_SCREENSHOTS](../ECLIPSE_SCREENSHOTS.md) for screenshots that demonstrate how to get and install the eclipseFormatFile and importOrderFile mentioned above.
 
 <a name="scala"></a>
 

